the-collab-lab/tcl-25-smart-shopping-list
# Created list view and add an item view

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description
This PR sets up React Router and creates two new components: one for viewing the list and another for adding new items.



## Related Issue

Closes #3 

## Acceptance Criteria

- react-router-dom has been added as a project dependency
- Links are present and persistent at the bottom of the app: one for the “list” view, the other for the “add an item” view
- When one of the links is clicked, the browser URL updates to represent the current view and a matching view component is displayed
- Whichever view is selected, the corresponding link should display in bold text

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After




## Testing Steps / QA Criteria
From your terminal, pull down this branch with `git pull origin ci-cz-react-router` and check out the branch with `git checkout ci-cz-react-router`.
Run `npm install` to install the newly added dependency and then `npm start` to launch the app. You should view the updates in your browser.
